docs/rst/manual/about/release_notes.rst

```diff
@@ -27,6 +27,96 @@ Release Notes
  Miscellaneous:
    -
 
+Version 2.35
+------------
+
+Breaking changes:
+  - Change default inversion to IES_INVERSION_EXACT (`#3193 <https://github.com/equinor/ert/pull/3193>`_)
+  - Remove flag for using aa_projection in IES and ES (`#3230 <https://github.com/equinor/ert/pull/3230>`_)
+  - Fix scaling of ESMDA weights (`#3211 <https://github.com/equinor/ert/pull/3211>`_)
+  - Replaced fm message with logging statements and remove unused workflows LOAD_RESULTS(_ITER) (`#3252 <https://github.com/equinor/ert/pull/3252>`_)
+  - Remove option of loading from non-unified summary files (`#3247 <https://github.com/equinor/ert/pull/3247>`_)
+  - Remove setting `MODULE` from workflows (`#3288 <https://github.com/equinor/ert/pull/3288>`_)
+  - Remove analysis enums (`#3283 <https://github.com/equinor/ert/pull/3283>`_)
+
+Bugfixes:
+  - Fix AnalysisIterConfig._repr_() and add test (`#3171 <https://github.com/equinor/ert/pull/3171>`_)
+  - Fix index out of bounds for active realizations < ensemble size in ESMDA (`#3200 <https://github.com/equinor/ert/pull/3200>`_)
+  - Fix bug where valid run_path was not recognised (`#3254 <https://github.com/equinor/ert/pull/3254>`_)
+
+New features:
+  - ert3: Add realization selection support in ert3 (`#3095 <https://github.com/equinor/ert/pull/3095>`_)
+  - ert3: Add visualise parameter to ert3 for starting webviz-ert (`#3209 <https://github.com/equinor/ert/pull/3209>`_)
+  - ert3: Add support for a scalar NumericalRecord (`#2934 <https://github.com/equinor/ert/pull/2934>`_)
+
+Improvements:
+  - Update poly_example to use more accurate observations (`#3149 <https://github.com/equinor/ert/pull/3149>`_)
+  - Upgrade ERT icon set to Equinor design system (`#3178 <https://github.com/equinor/ert/pull/3178>`_)
+  - Drop Title Case In Ert Gui (`#3190 <https://github.com/equinor/ert/pull/3190>`_)
+  - Remove .index files from block_fs (`#3185 <https://github.com/equinor/ert/pull/3185>`_)
+  - Add event viewer tool to gui (`#3136 <https://github.com/equinor/ert/pull/3136>`_)
```

Comment:
I _think_ I added the `enhancement` label before the switch to: `release-notes:` labels, but maybe I selected the wrong one - anyway, this is arguably a new feature.

Reply:
It had the release notes improvement label, but I changed it now :slightly_smiling_face: